components/CookieBanner.vue

```diff
@@ -1,66 +1,106 @@
-          <div
-            class="flex items-center justify-between md:!my-0 w-full md:w-fit">
-            <NeoButton variant="text" no-shadow @click="declineCookies">
-              {{ $t('cookies.decline') }}
-            </NeoButton>
-            <NeoButton
-              variant="secondary-rounded"
-              no-shadow
-              class="ml-3 h-8"
-              data-testid="cookie-banner-button-accept"
-              @click="acceptCookies">
-              {{ $t('cookies.accept') }}
-            </NeoButton>
-          </div>
-        </div>
-      </div>
-    </div>
+    <CookieControl class="cookie-banner">
+      <template #bar>
+        <p>
+          {{ $t('cookies.notice') }}
+          <nuxt-link to="/cookie-notice" target="_blank" class="is-underlined"
+            >Cookie Policy</nuxt-link
+          >
+          <span class="invisible md:visible mx-2">|</span>
+        </p>
+      </template>
+    </CookieControl>
   </transition>
 </template>
 
 <script lang="ts" setup>
-import { NeoButton } from '@kodadot1/brick'
-// import { useState } from 'vue-gtag-next'
+const cookieControl = useCookieControl()
+const { grantConsent, revokeConsent } = useGtag()
 
-// const { isEnabled } = useState()
-const hasDisplayedCookieBanner = ref(
-  localStorage.getItem('cookies_enabled') !== null || false,
+const cookieConsentGiven = computed(() =>
+  cookieControl.cookiesEnabledIds.value?.includes('ga'),
 )
-const acceptCookies = () => {
-  // if (isEnabled) {
-  //   isEnabled.value = true
-  // }
-  localStorage.setItem('cookies_enabled', '1')
-  hasDisplayedCookieBanner.value = true
-}
-const declineCookies = () => {
-  localStorage.setItem('cookies_enabled', '0')
-  hasDisplayedCookieBanner.value = true
-}
+
+watch(
+  () => cookieControl.cookiesEnabledIds.value,
+  () => {
+    if (!cookieConsentGiven.value) {
+      window.location.reload()
+    }
+  },
+  { deep: true },
+)
+
+onMounted(() => {
+  if (cookieConsentGiven.value || cookieConsentGiven.value === undefined) {
+    grantConsent()
+  } else {
+    revokeConsent()
+  }
+})
 </script>
 
 <style lang="scss">
-.cookie-banner {
-  .snackbar {
-    align-self: flex-start;
-    max-width: 720px;
+/* Custom Cookie #Bar */
+.cookieControl__BarContainer {
+  @apply fixed p-4 mx-auto max-w-3xl self-start left-6 bottom-5 text-text-color bg-background-color border border-border-color shadow-primary;
+}
+.cookieControl__BarContainer p {
+  @apply text-text-color text-base;
+}
+.cookieControl__BarButtons button {
+  @apply text-sm bg-inherit text-text-color px-2 py-1;
+}
+
+/* Custom Cookie #Modal */
+.cookieControl__ModalContent {
+  @apply text-text-color bg-background-color border border-border-color shadow-primary items-center overflow-y-clip;
+}
+.cookieControl__ModalContent :not(button) {
+  @apply text-text-color;
+}
+button.cookieControl__ModalClose {
+  font-family: 'Font Awesome 6 Sharp';
```

Comment:
Could we remove the `font family`?
![image](https://github.com/kodadot/nft-gallery/assets/31397967/65444ae5-1a26-4d5c-994b-e2c2ef8faa7e)

Reply:
✔️